### DISCUSSION TITLE
Welcome to adXLauncher Discussions!

### DISCUSSION THREAD

**Discussion by DustinMorgan** [SEVENNetworks/adX-Launcher-Open-Source-Issue-Tracker#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with our userbase. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with the team and other users.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, join the Alpha program by:
1. Joining our [Google Group](https://groups.google.com/g/mx-alpha) to get access to the program
2. Click this link to [become an Alpha tester](https://play.google.com/apps/testing/com.seven.mx.launcher)

**Comment by Ahmdmslb238** [comment]
عربي @DustinMorgan

**Reply by Ahmdmslb238** [reply]
انا اريد ان  اتعلم لغة انجليزية @DustinMorgan  ولكن لم أجد 🌖

**Reply by Ahmdmslb238** [reply]
مرحبا 👋

**Reply by DustinMorgan** [reply]
Welcome! Unfortunately, we currently only offer support in English, but we will do our best with an online translator. 

We hope to support more languages in the app soon.

**Comment by Ed1i** [comment]
Hello, is it possible to make AdxLauncher work on Android 13 ?

**Comment by KING07-svg** [comment]
Cxdhmyktd5u
Ya1

**Comment by joesphking920-stack** [comment]
Good

**Comment by joesphking920-stack** [comment]
- []()[]()@

**Comment by joesphking920-stack** [comment]
- ,,~~@ ~~

**Comment by joesphking920-stack** [comment]
# -

**Comment by chovizking33** [comment]
Holpa
- - [ ] Hola

**Comment by chovizking33** [comment]
https://github.com/OpenBMB/VoxCPM/commit/68af4fe502d63acaa80de31261c835fb3165e333

**Comment by plooolp** [comment]
> ## 👋 Welcome!
>   We’re using Discussions as a place to connect with our userbase. We hope that you:
>   * Ask questions you’re wondering about.
>   * Share ideas.
>   * Engage with the team and other users.
>   * Welcome others and are open-minded. Remember that this is a community we
>   build together 💪.
> 
>   To get started, join the Alpha program by:
> 1. Joining our [Google Group](https://groups.google.com/g/mx-alpha) to get access to the program
> 2. Click this link to [become an Alpha tester](https://play.google.com/apps/testing/com.seven.mx.launcher)

**Comment by plooolp** [comment]
👋 Welcome! 
 We’re using Discussions as a place to connect with our userbase. We hope that you: 
 
 Ask questions you’re wondering about. 
 Share ideas. 
 Engage with the team and other users. 
 Welcome others and are open-minded. Remember that this is a community we 
 
build together 💪. 
 
 To get started, join the Alpha program by: 
 
 Joining our  Google Group  to get access to the program 
 Click this link to  become an Alpha tester
